homeassistant/components/intent_script/__init__.py

```diff
@@ -55,9 +62,25 @@
 )
 
 
-async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
-    """Set up the intent script component."""
-    intents = config[DOMAIN]
+async def async_reload(
+    hass: HomeAssistant, servie_call: ServiceCall, existing_intents: dict
+) -> None:
+    """Handle start HomeKit service call."""
+    new_config = await async_integration_yaml_config(hass, DOMAIN)
+
+    if not new_config or DOMAIN not in new_config:
+        return
```

Comment:
move this underneath the for to remove it all

Reply:
Done